src/components/ContactForm/index.js

```diff
@@ -130,15 +130,15 @@ const ContactForm = () => {
               }
               onInput={(e) => e.target.setCustomValidity("")}
             />
-            <label htmlFor="ScopeOfQuestion" className="form-name">
+            <label htmlFor="scope" className="form-name">
```

Comment:
I don't know if this will negatively affect the MailChimp form behind this or not. You might submit a test contact request to confirm.

Reply:
No, this is not connected to the MailChimp contact form. I just ran a test, it worked without any issue.